servicex/servicex_adapter.py

```diff
@@ -162,6 +163,22 @@ async def get_servicex_info(self) -> ServiceXInfo:
     async def get_servicex_capabilities(self) -> List[str]:
         return (await self.get_servicex_info()).capabilities
 
+    async def get_servicex_sample_title_limit(self) -> Optional[int]:
+        # check if the capability is defined
+        capabilities = await self.get_servicex_capabilities()
+        for capability in capabilities:
+            if capability.startswith("long_sample_titles_"):
+                try:
+                    # hope capability is of the form long_sample_titles_NNNN
+                    return int(capability[19:])
+                except ValueError:
+                    raise RuntimeError(
```

Comment:
If we have SX change how it delivers this, this will make it impossible to run against this SX. Perhaps, if there is an exception, then just move on, and return None if it isn't in the list of capabilities?

Reply:
Use `logging.warn` or similar?